dashboard/signals.py

```diff
@@ -101,6 +75,8 @@ def handle_delete_certificate(sender, instance, **kwargs):  # pylint: disable=un
     """
     user = instance.user
     program = instance.course_run.course.program
-    program_enrollment = ProgramEnrollment.objects.filter(user=user, program=program).first()
-    if program_enrollment is not None:
+    try:
+        program_enrollment = ProgramEnrollment.objects.get(user=user, program=program)
         index_program_enrolled_users.delay([program_enrollment])
+    except ProgramEnrollment.DoesNotExist:
+        pass
```

Comment:
I'm not sure that would help with clarity here. The try except is a standard way of testing if objects exist in Django. The helper function would not be adding much and it wouldn't be as obvious to people who were familiar with Django but not our code base